client/components/map/index.jsx

```diff
@@ -32,6 +32,7 @@ const map = ({
     zoom={zoom}
     onLeafletMoveend={(ev) => actions.map.getGeoJSON(ev)}
     onLeafletResize={(ev) => actions.map.getGeoJSON(ev)}
+    onLeafletClick={() => actions.map.setActive(undefined)}
```

Comment:
Does the popup not have a "x"? I know our side list currently lacks this, but especially if we are wanting to move towards being able to do multi-select, the pop up may be blocking the view of another trail. An explicit clear seems like a better idea to me..